src/main/java/com/hollingsworth/arsnouveau/common/event/timed/RewindEvent.java

```diff
@@ -86,6 +100,18 @@ public void onServerStopping() {
         }
     }
 
+    public void onEntityRemoved(PlayerEvent.PlayerLoggedOutEvent event) {
+        if (entity == null) {
+            NeoForge.EVENT_BUS.unregister(this);
+            return;
+        }
+
+        if (entity == event.getEntity() && event.getEntity().isNoGravity()) {
+            event.getEntity().setNoGravity(false);
```

Comment:
I think we would need to check if they were affected by gravity at the start, otherwise this will disable gravity from mobs that were originally noGravity = true.